taxonomy.simple.yaml

```diff
@@ -54,209 +43,154 @@
+        - University Prep
       - Tutoring
       - Youth Development
       - Youth Movement
     - Leadership development
-    - Preschool
     - Screening & Exams:
-      - English as a Second Language (ESL)
-      - GED/High-School Equivalency
+      - Bagrut/High-School Equivalency
     - Skills & Training:
       - Basic Literacy
       - Computer Class
       - Daily Life Skills
       - Interview Training
       - Skills Assessment
       - Specialized Training
-  
+
   - Emergency:
     - Disaster Response
     - Emergency Food
     - Emergency Helplines
-    - Emergency Payments:
-      - Help Pay for Food
-      - Help Pay for Healthcare
-      - Help Pay for Housing
-      - Help Pay for School
     - Emergency Shelter
     - Help Find Missing Persons
-    - Immediate Safety:
-      - Help Escape Violence
-      - Safe Housing
     - Psychiatric Emergency Services
-  
+
   - Food:
-    - Community Gardens
-    - Emergency Food
     - Food Delivery
     - Food Pantry
-    - Help Pay for Food:
-      - Government Food Benefits
     - Nutrition
     - Nutrition Education
-  
+
   - Goods:
-    - Baby Supplies:
-      - Baby Clothes
-      - Diapers & Formula
-    - Clothing:
-      - Clothes for School
-      - Clothes for Work
-      - Clothing Vouchers
-    - Home Goods:
-      - Blankets
-      - Books
-      - Efficient Appliances
-      - Fans
-      - Furniture
-      - Personal Care Items
-      - Supplies for School
-      - Supplies for Work
-    - Medical Supplies:
-      - Assistive Technology
-      - Prosthesis
-    - Personal Safety
+    - Baby Supplies
+    - Clothing
+    - Home Goods
     - Toys & Gifts
-  
+
   - Health:
     - Addiction & Recovery:
       - Detox
       - Drug Testing
       - Medications for Addiction
-      - Outpatient Treatment
       - Peer Recovery Coaching
       - Residential Treatment
       - Substance Abuse Counseling
     - Dental Care
     - Emergency Services:
       - Psychiatric Emergency Services
       - Dental Emergency Service
-      - Exercise & Fitness
     - Help Find Healthcare
-    - End-of-Life Care:
-      - Bereavement
-      - Burial & Funeral Help
-      - Hospice
-      - Pain Management
+    - End-of-Life Care
     - Health Education:
       - Disease Management
       - Family Planning
       - Nutrition Education
-      - Parenting Education
       - Sex Education
-    - Help Pay for Healthcare:
-      - Disability Benefits
-      - Discounted Healthcare
-      - Health Insurance
-      - Medical Supplies
-      - Prescription Assistance
-      - Transportation for Healthcare
-    - Medical Care:
+    - Medical care:
       - Allied Health
       - Alternative Medicine
       - Checkup & Test:
-        - Checkup & Test
-        - Disability Screening
         - Disease Screening
         - Eye Tests
         - Genetic Testing
         - Hearing Tests
         - Pregnancy Tests
-      - Combined Care
-      - Maternity Care
+        - Maternity Care
```

Comment:
Maternity care was indented by mistake I think (it wasn't inside checkup & test before)